storytower/src/components/Header.jsx

```diff
@@ -11,12 +11,12 @@ const Header = () => {
   }
   // Not a comprehensive list of genres. I don't know how to better check this, I was thinking of using a mongodb query that finds all the unique values that are in the genres field of a bunch of different stories maybe?
```

Comment:
that's how I would do it - or any time a new story is added check to see if it's genre exists on list of known genres and add if not present